gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerRequestManager.java

```diff
@@ -46,17 +49,22 @@
   private final Duration requestTimeout;
 
   public BrokerRequestManager(
-      ClientOutput clientOutput,
-      BrokerTopologyManagerImpl topologyManager,
-      RequestDispatchStrategy dispatchStrategy,
-      Duration requestTimeout) {
+      final ClientOutput clientOutput,
+      final BrokerTopologyManagerImpl topologyManager,
+      final RequestDispatchStrategy dispatchStrategy,
+      final Duration requestTimeout) {
     this.clientOutput = clientOutput;
     this.dispatchStrategy = dispatchStrategy;
     this.topologyManager = topologyManager;
     this.requestTimeout = requestTimeout;
   }
 
-  private static boolean shouldRetryRequest(final DirectBuffer responseContent) {
+  private static boolean shouldRetryRequest(final IncomingResponse response) {
```

Comment:
Can we rename this so it's more explicit where/when it should be used? Something like
```suggestion
  private static boolean shouldRetryFixedPartitionRequestfinal IncomingResponse response) {
```

Or something similar